views/scss/qti/_qti-base.scss

```diff
@@ -84,20 +84,20 @@ $mainContainer : '.qti-item';
     padding: 15px;
     background: white();
     margin:auto;
-    max-width: 1024px;
+    max-inline-size: 1024px;
 
     &.runtime {
-        min-height: 100%;
+        min-block-size: 100%;
     }
 
     [class*="col-"] {
         img {
-            max-width: initial;
+            max-inline-size: initial;
```

Comment:
Note: 
for images, we'll need something more clever: https://github.com/oat-sa/tao-deliver-fe/pull/1352/commits/b2e85f6e3f18a85b9db54bf04776a42429216750
Will add later in https://github.com/oat-sa/extension-tao-itemqti/pull/2669